class-two-factor-compat.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * A compatability layer for some of the most popular plugins.
+ *
+ * Should be used with care because ideally we wouldn't need
+ * any integration specific code for this plugin. Everything should
+ * be handled through clever use of hooks and best practices.
+ */
+class Two_Factor_Compat {
+	/**
+	 * Initialize all the custom hooks as necessary.
+	 *
+	 * @return void
+	 */
+	public function init() {
+		/**
+		 * Jetpack
+		 *
+		 * @see https://wordpress.org/plugins/jetpack/
+		 */
+		add_filter( 'two_factor_rememberme', array( $this, 'jetpack_rememberme' ) );
```

Comment:
Maybe add a condition to only attach the hook if Jetpack is installed?

Reply:
Were you thinking of checking for the presence of the `Jetpack` class or one of the constants `JETPACK__VERSION` or `JETPACK__PLUGIN_FILE`? The `is_plugin_active()` check could fail if Jetpack is installed in a different directory, for example (during dev or some custom plugin tree).

Reply:
I was thinking testing `Jetpack` class presence